front-end/src/app/users/users.module.ts

```diff
@@ -0,0 +1,43 @@
+// Angular
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+// PrimeNG
+import { TableModule } from 'primeng/table';
+import { DropdownModule } from 'primeng/dropdown';
+import { ToastModule } from 'primeng/toast';
+import { ToolbarModule } from 'primeng/toolbar';
+import { FileUploadModule } from 'primeng/fileupload';
+import { ButtonModule } from 'primeng/button';
+import { CheckboxModule } from 'primeng/checkbox';
+import { ConfirmDialogModule } from 'primeng/confirmdialog';
+import { DialogModule } from 'primeng/dialog';
+import { InputTextModule } from 'primeng/inputtext';
```

Comment:
A number of these PrimeNG modules, such as FileUploadModule, are not used by the Users module and can be removed. You will know it isn't used if you remove the import line and import declaration below and the app recompiles without error

Reply:
I've removed all of the PrimeNG imports and added them back one-by-one as was necessary to get the code to compile.  There should be no more unnecessary imports present.